### DISCUSSION TITLE
Possibility to change default view

### DISCUSSION THREAD

**Discussion by fdw** [jonaswinkler/paperless-ng#761] [Feature Requests]
Is there a way to define the default view in regards to filters and sorting?

Usually, after I added a document, I want to check if all the metadata is correct. But if I just open "documents", the default sorting is "created", and often documents have the wrong date (which is why I want to check the metadata in the first place).
I can define a new view, but I'd rather just change the default one.

**Comment by jonaswinkler** [comment]
Hello!

The default list view remembers the filter and sorting settings for the duration of the current browsing session.

1. It's easy to change that so that your browser will remember sorting and filtering settings across browsing sessions, but these settings won't be shared between multiple browsers / machines.
2. It's somewhat more difficult to store that on the server.

Maybe option 1 is already enough?

**Comment by shamoon** [comment]
Also for your specific use case perhaps Inbox tags are useful? Same thing happens to me if the date is wrong the document is buried but will still be the only one(s) in the inbox this easy to find.
